### PR TITLE
locale-source: translations added

### PR DESCRIPTION
**Pull request by m4ra**
**Tasks**
- [ ] PR name contains story or task reference
- [ ] Documentation (docs and inline)
- [ ] Tests (including n+1 and django_assert_num_queries where applicable)
- [ ] Changelog

### CODE REVIEW

**Comment by m4ra** [comment]
@TeaJayyy I have pushed translations to transifex, please have a look.

**Review by goapunk — APPROVED** [review]
LGTM

**Comment by TeaJayyy** [comment]
@m4ra @goapunk done

File: locale-source/locale/de/LC_MESSAGES/djangojs.po [hunk 1]
```diff
@@ -234,13 +232,13 @@ msgstr ""
 #: adhocracy4/comments_async/static/comments_async/ai_report.jsx:13
 #: adhocracy4/comments_async/static/comments_async/ai_report.jsx:13
 msgctxt "defakts"
-msgid "Show info to users"
+msgid "Show AI info to users"
```
Comment (m4ra) [inline locale-source/locale/de/LC_MESSAGES/djangojs.po:235]: @TeaJayyy the defakts translations come from A4. Need to remove them manually in the future before pushing to transifex.

**Pull request closed without merging**